### PR TITLE
ZFIN-9244 Fix load Start / End of NCBI genes

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: server_apps/data_transfer/NCBIStartEnd/NCBIStartEnd.pl [hunk 1]
```diff
@@ -110,14 +112,14 @@
...
+                    from db_link, foreign_db_contains
+                   where dblink_fdbcont_zdb_id = fdbcont_zdb_id
                      and fdbcont_fdb_db_id = 10
-                     and not exists(select 1 from sequence_feature_chromosome_location_generated 
-                                     where sfclg_data_zdb_id = dblink_linked_recid 
-                                       and sfclg_location_source = 'NCBIStartEndLoader' 
-                                       and sfclg_fdb_db_id = 10);";           
+                     and not exists(select 1 from sequence_feature_chromosome_location_generated
+                                     where sfclg_data_zdb_id = dblink_linked_recid
+                                       and sfclg_locatoni_source = 'NCBIStartEndLoader'
```
Comment (rtaylorzfin) [inline server_apps/data_transfer/NCBIStartEnd/NCBIStartEnd.pl:121]: locatoni looks like a typo

File: source/org/zfin/db/postGmakePostloaddb/1160/db.changelog.master.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <include file="source/org/zfin/db/postGmakePostloaddb/1159/ZFIN-9244.sql" />
```
Comment (rtaylorzfin) [inline source/org/zfin/db/postGmakePostloaddb/1160/db.changelog.master.xml:8]: 1159 should be 1160?

File: server_apps/data_transfer/NCBIStartEnd/NCBIStartEnd.pl (hunk 1 above)
Comment (cmpich) [reply server_apps/data_transfer/NCBIStartEnd/NCBIStartEnd.pl]: Good catch.

File: source/org/zfin/db/postGmakePostloaddb/1160/db.changelog.master.xml (hunk 2 above)
Comment (cmpich) [reply source/org/zfin/db/postGmakePostloaddb/1160/db.changelog.master.xml]: Corrected it.

**Review by rtaylorzfin — APPROVED** [review]

**Pull request merged**